ui/raidboss/data/04-sb/raid/o4s.js

```diff
@@ -953,26 +918,7 @@
       condition: function(data, matches) {
         return matches.target == data.me;
       },
-      alarmText: function(data) {
-        if (data.role != 'tank') {
-          return {
-            en: 'Earthshaker on YOU',
-            de: 'Erdstoß auf DIR',
-          };
-        }
-      },
-      infoText: function(data) {
-        if (data.role == 'tank') {
-          return {
-            en: 'Earthshaker on YOU',
-            de: 'Erdstoß auf DIR',
-          };
-        }
-      },
-      tts: {
-        en: 'shaker',
-        de: 'erstoß',
-      },
+      response: Responses.earthshaker(),
```

Comment:
I think this loses the alarmText/infoText nature here.  Feel free to split this up into two triggers with different conditions, but letting non-tanks (who don't expect earthshakers) get an alarm is important, I think.

Reply:
ok i gave it a try, hope it is correct